ParquetHadoop/src/main/java/io/deephaven/parquet/base/tempfix/ParquetMetadataConverter.java

```diff
@@ -1,1538 +0,0 @@
-/*
- * Licensed to the Apache Software Foundation (ASF) under one
- * or more contributor license agreements.  See the NOTICE file
- * distributed with this work for additional information
- * regarding copyright ownership.  The ASF licenses this file
- * to you under the Apache License, Version 2.0 (the
- * "License"); you may not use this file except in compliance
- * with the License.  You may obtain a copy of the License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing,
- * software distributed under the License is distributed on an
- * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
- * KIND, either express or implied.  See the License for the
- * specific language governing permissions and limitations
- * under the License.
- */
-package io.deephaven.parquet.base.tempfix;
-
-/*
-TODO(deephaven-core#901): Remove the hacked ParquetMetadataConverter.java and the need for the ParquetHadoop module.
-NOTE this only exists for this line, inside addRowGroup
-Without it the page offset is not being saved properly
-            if (columnMetaData.getDictionaryPageOffset() >= 0) {
-                columnChunk.meta_data.setDictionary_page_offset(columnMetaData.getDictionaryPageOffset());
-            }
```

Comment:
I'm assuming this is now handled by the underlying library?

Very happy to merge assuming the above is true, or no longer relevant.

Reply:
Hi Devin, Andy told me that he confirmed from Apache repo that the all the underlying issues have been fixed.

Reply:
This is the [bug fix commit](https://github.com/apache/parquet-mr/commit/0acade062893cc0ac6e8cb7ed01470c38ea4bdb5). This fix was released in parquet-hadoop v1.12.0. Core is already on v1.13.0.